packages/tooltip/src/vaadin-tooltip.js

```diff
@@ -449,8 +471,8 @@ class Tooltip extends ThemePropertyMixin(ElementMixin(PolymerElement)) {
     this.__isTargetHidden = !isVisible;
 
     // Open the overlay when the target is visible and has focus or hover.
-    if (isVisible && (this.__focusInside || this.__hoverInside)) {
-      this._open(true);
+    if (isVisible && (this.__focusInside || this.__hoverInside) && !this._autoOpened) {
```

Comment:
The new check feels unnecessary, but without it some test was failing. 
Apparently now it works after I added more delays in `beforeEach`.